App.tsx

```diff
@@ -1,13 +1,11 @@
 import { StatusBar } from "expo-status-bar";
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
+import StackNavigator from "./ui/screens/login/RootStack";
 
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+    <StackNavigator/>
```

Comment:
This is great for testing, but now that the work is done you can change it back to the default.